Source/WebCore/rendering/style/RenderStyleInlines.h

```diff
@@ -671,6 +672,7 @@ inline float RenderStyle::textStrokeWidth() const { return m_rareInheritedData->
 inline OptionSet<TextTransform> RenderStyle::textTransform() const { return OptionSet<TextTransform>::fromRaw(m_inheritedFlags.textTransform); }
 inline TextUnderlineOffset RenderStyle::textUnderlineOffset() const { return m_rareInheritedData->textUnderlineOffset; }
 inline TextUnderlinePosition RenderStyle::textUnderlinePosition() const { return static_cast<TextUnderlinePosition>(m_rareInheritedData->textUnderlinePosition); }
+inline WhiteSpaceCollapse RenderStyle::whiteSpaceCollapse() const { return static_cast<WhiteSpaceCollapse>(m_rareInheritedData->whiteSpaceCollapse); }
```

Comment:
Discussed this with others, and this should be on `m_inheritedData` since the `white-space` property is frequently set (meaning `white-space-collapse` & `text-wrap` would be as well). `text-wrap` is currently on `m_rareInheritedData`, but that's a mistake.

Reply:
I don't quite know the difference between `m_inheritedData` and `m_inheritedFlags`, but it seems like the `white-space` property is implemented in `m_inheritedFlags`. Do you have an idea on how to discern which properties go where?